image_LICENSE.txt

```diff
@@ -12,10 +12,3 @@ Unless stated in this file, icons are work of Enthought, and are released
 under BSD-like license.
 Files and original authors:
 ----------------------------------------------------------------
```

Comment:
```suggestion

```

Reply:
and you can remove `GV: ...` from the earlier `Project/License/File` list.

Reply:
this can be done in a separate PR but it looks like the image license file needs to be updated. After this PR, it looks like there will be no images left in `apptools`. can you open a separate issue for this please?

Reply:
opened #236